packages/chord-mark/tests/unit/renderer/components/renderSong.spec.js

```diff
@@ -690,3 +690,48 @@ describe('Custom ChordSymbol renderer', () => {
 		expect(toText(rendered)).toBe('|Custom     |Custom     |Custom     |');
 	});
 });
+
+describe('Wrap Sections in Divs', () => {
+	test('Correctly create and class sections', () => {
+		const song = `4/4
+#v
+A B
+verseLine1
+C.. D.. E
+verseLine2
+
+#v
+A B
+verseLine1
+
+#c
+A B
+chorusLine1
+
+#b
+A B
+brigeLine1
+
+#c`;
+		const expectedSectionClasses = [
+			'cmSection-Verse',
+			'cmSection-Verse',
+			'cmSection-Chorus',
+			'cmSection-Bridge',
+			'cmSection-Chorus'
+		];
+		const rendered = renderSongText(song);
+		const element = htmlToElement(rendered);
+
+		expect(element.childElementCount).toBe(6);
+		
+		element.childNodes.forEach((child, i) => {
```

Comment:
Same thing here, having conditional tests is considered bad practice since the logic might contains flaws, yield false negative/positive and make the test harder to read. Don't be afraid to be a bit verbose. Of course the test code is still code that has to be maintained and as such coding good practices also apply to test code but if the tests are small and atomic enough then a bit of repetition should not hurt readability / maintainability. 

Generally speaking the more straightforward and the less logic you have in tests, the better. 

In practice:
```javascript
expect(element.childNodes[0].nodeName).toBe('P');
expect(element.childNodes[1].nodeName).toBe('DIV');
expect(element.childNodes[1].className).toBe('cmSection cmSection-Verse');
expect(element.childNodes[2].nodeName).toBe('DIV');
expect(element.childNodes[2].className).toBe('cmSection cmSection-Verse');
...
```